arkouda/pdarraycreation.py

```diff
@@ -728,7 +750,7 @@ def random_strings_uniform(minlen : int, maxlen : int, size : int,
 
```

Comment:
cast again? is it needed?

Reply:
@mhmerrill yes, generic_msg returns str or bytes